docs/en/docs/ghippo/user-guide/access-control/global.md

```diff
@@ -1,39 +1,44 @@
-# Global management mode
+# System Roles
 
 ## Applicable scenarios
 
-If you need to set up administrators for sub-modules to further manage user/group permissions through administrators, you can use roles in Access Control. Possessing this role means having the highest administrative authority for this submodule.
+DCE 5.0 provides predefined system roles to help users simplify the process of role permission usage.
 
 !!! note
 
-    There are three modes of permission management, currently the global management mode. The role in the access control represents the highest authority of each sub-module. See [Access Management](role.md) for details.
+    DCE 5.0 provides three types of system roles: platform role, workspace role, and folder role.
+    Platform role: has corresponding permissions for all related resources on the platform. Please go to user/user group authorization.
```

Comment:
"Please go to user/user group authorization."

May need a little bit improvement (e.g., add links ) to make it easier to understand, as well as the similar sentences in Line 11 and 12